doc/doxygen/Makefile

```diff
@@ -1,7 +1,7 @@
 RIOTBASE=$(shell git rev-parse --show-toplevel)
 # Generate list of quoted absolute include paths. Evaluated in riot.doxyfile.
 export STRIP_FROM_INC_PATH_LIST=$(shell \
-    git ls-tree -dr --full-tree --name-only HEAD |\
+    git ls-tree -dr --full-tree --name-only HEAD core drivers sys |\
```

Comment:
Proper documentation of packages is on my todo list. I think RIOT's documentation should at least indicate which packages are available and outline the main purpose of each package.